packages/medusa-payment-paypal/src/core/paypal-sdk.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  CreateOrder,
+  CreateOrderResponse,
+  GetOrderResponse,
+  PatchOrder,
+  PaypalApiPath,
+  PaypalSdkOptions,
+} from "./types"
+import {
+  CapturesAuthorization,
+  CapturesAuthorizationResponse,
+  CapturesRefund,
+  CapturesRefundResponse,
+} from "./types/payment"
+import { PaypalHttpClient } from "./paypal-http-client"
+import { VerifyWebhookSignature } from "./types/webhook"
+
+export class PaypalSdk {
```

Comment:
comment: I know, you've probably followed the official SDK, but I would change the method names slightly. Will add my suggestions. Feel free to discard some or all of them.